demo/mkProfiles.py

```diff
@@ -0,0 +1,180 @@
+###############################################
+# Make user profiles by adding document vectors
+###############################################
+
+import sys
+import numpy as np
+from scipy.spatial import distance
+import os
+
+path_to_PeARS = os.path.join(os.path.dirname(__file__),"users/")
+stopwords=["","i","a","about","an","and","each","are","as","at","be","are","were","being","by","do","does","did","for","from","how","in","is","it","its","make","made","of","on","or","s","that","the","this","to","was","what","when","where","who","will","with","has","had","have","he","she","one","also","his","her","their","only","both","they","however","then","later","but","never","which","many"]
+num_dimensions=400
+dm_dict={}
+
+###############################################
+# Read list of users
+###############################################
+
+def readUsers(usernames_file):
+	#print "Getting users..."
+	users=[]
+	f=open(usernames_file,'r')
+	for l in f:
+		l=l.rstrip('\n')
+		users.append(l)
+	f.close()
+	return users
+
+#############################################
+# Normalisation
+#############################################
+
+def normalise(v):
```

Comment:
@minimalparts The code looks beautiful! We will have to DRY it up a bit later. Looks like the logic here repeats in mkWordClouds. Making a mental note to make it common later.